src/components/signIn/SignIn.tsx

```diff
@@ -1,42 +1,83 @@
-import React from 'react'
 import { Link } from 'react-router-dom'
+import styles from '../../pages/Auth/Auth.module.css'
 import './SignIn.css'
+import { logInSchema } from '../../pages/Auth/Validation'
+import {useFormik} from 'formik'
+
+interface FormValues {
+  username: string;
+  password: string;
+}
 
 function SignIn() {
+
+  const initialValues: FormValues = {
+    username: '',
+    password: '',
+  };
+
+  const onSubmit = (values: FormValues) => {
+    console.log(values)
+  };
+
+  const formik = useFormik<FormValues>({
+    initialValues,
+    onSubmit,
+    validationSchema: logInSchema,
+  });
+
+
   return (
-    <div className="a-right">
-        <form className="infoForm authForm">
-          <h3>Log In</h3>
-  
-          <div>
+    <div className={styles['right-section']}>
```

Comment:
don't use - in classnames to e able to use the dot property accessor way. styles.rightSection

Reply:
ok